### ISSUE TITLE
Sql Server Management Studio fails to remove the installer.

### ISSUE THREAD

**Issue by Bartmax** [microsoft/winget-cli#428]
trying to install Sql Server Management Studio:
https://github.com/microsoft/winget-pkgs/blob/master/manifests/Microsoft/SQLServerManagementStudio/18.5.yaml

fails to remove the installer
```
microsoft/winget-pkgs#10 [Bart]: winget install "sql management studio"
Found SQL Server Management Studio [Microsoft.SQLServerManagementStudio]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://go.microsoft.com/fwlink/?linkid=2125901&clcid=0x409
  ██████████████████████████████   534 MB /  534 MB
Successfully verified installer hash
Installing ...
Successfully installed!
An unexpected error occurred while executing the command:
remove: Access is denied.: "C:\Users\Bart\AppData\Local\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\TempState\WinGet\Microsoft.SQLServerManagementStudio.18.5.exe"
microsoft/winget-pkgs#11 [Bart]:
```

**Comment by denelon** [comment]
https://github.com/microsoft/winget-cli/issues/418

**Comment by denelon** [comment]
This bug will be fixed when we implement #418

**Comment by denelon** [comment]
Duplicate of #418
